$NetBSD: patch-Makefile,v 1.2 2017/07/03 11:53:57 wiedi Exp $

Do not pick pkgsrc libtool on Darwin.
NetBSD needs _NETBSD_SOURCE to be defined.
Use correct installation dirs.

--- Makefile.orig	2016-02-09 09:38:56.000000000 +0000
+++ Makefile
@@ -15,7 +15,7 @@ VERSION = $(shell cat VERSION)
 
 ifeq ($(OS), Darwin)
 JARSUFFIX=mac
-LIBTOOL ?= $(shell which libtool)
+LIBTOOL ?= /usr/bin/libtool
 endif
 ifeq ($(OS), Linux)
 JARSUFFIX=linux
@@ -38,6 +38,9 @@ CFLAGS += -DMINIUPNPC_SET_SOCKET_TIMEOUT
 CFLAGS += -DMINIUPNPC_GET_SRC_ADDR
 CFLAGS += -D_BSD_SOURCE
 CFLAGS += -D_DEFAULT_SOURCE
+ifeq ($(OS), NetBSD)
+CFLAGS += -D_NETBSD_SOURCE
+endif
 ifneq ($(OS), FreeBSD)
 ifneq ($(OS), Darwin)
 #CFLAGS += -D_POSIX_C_SOURCE=200112L
@@ -136,11 +139,11 @@ endif
 
 LIBDIR ?= lib
 # install directories
-INSTALLPREFIX ?= $(PREFIX)/usr
+INSTALLPREFIX ?= $(PREFIX)
 INSTALLDIRINC = $(INSTALLPREFIX)/include/miniupnpc
 INSTALLDIRLIB = $(INSTALLPREFIX)/$(LIBDIR)
 INSTALLDIRBIN = $(INSTALLPREFIX)/bin
-INSTALLDIRMAN = $(INSTALLPREFIX)/share/man
+INSTALLDIRMAN = $(INSTALLPREFIX)/$(PKGMANDIR)
 
 FILESTOINSTALL = $(LIBRARY) $(EXECUTABLES)
 ifneq ($(OS), AmigaOS)
